objectRecognitionSource/src/test/java/io/vantiq/extsrc/objectRecognition/imageRetriever/TestNetworkStreamRetriever.java

```diff
@@ -19,8 +19,8 @@
     NetworkStreamRetriever retriever;
     NoSendORCore source;
     
-    final String IP_CAMERA_URL = "http://153.142.207.158:80/-wvhttp-01-/GetOneShot?image_size=640x480&frame_count=1000000000";
-    
+    static final String IP_CAMERA_URL = "http://60.45.181.202:8080/mjpg/quad/video.mjpg";
```

Comment:
Why is this one different from `http://49.229.157.154:8080/mjpg/video.mjpg` used in all the later tests?

Reply:
No particularly good reason except that that's what was found at the time whatever the camera was started to fail.